### DISCUSSION TITLE
Registration as a student

### DISCUSSION THREAD

**Discussion by Rana2006** [community/community#50591] [GitHub Education]
Hi,

I am trying to register as a student according to the information I got from CareerFoundry GmbH which explained that I need to upload a jpeg screenshot of my CareerFoundry enrollment confirmation.

I don't know how to do that.

Regards,
Rana

**Comment by Gf22AZMA** [comment]
what do you want exactly

**Comment by Gf22AZMA** [comment]
what do you want exactly

**Comment by Rana2006** [comment]
How can I add my document?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Rana2006 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
